tier4_system_msgs/msg/DiagnosticSummary.msg

```diff
@@ -0,0 +1,9 @@
+# Each indicates a level of diagnostic_msgs/msg/DiagnosticStatus.
+builtin_interfaces/Time stamp
+byte stop
+byte autonomous
+byte local
+byte remote
+byte emergency_stop
+byte comfortable_stop
+byte pull_over
```

Comment:
Can't these be boolean values?

Reply:
Changed https://github.com/tier4/tier4_autoware_msgs/pull/96/commits/62c3dddade6c4bfd92117b5739df6ffcffa3a391